addons/mail/static/tests/discuss_app/sidebar_tests.js

```diff
@@ -633,13 +678,33 @@ QUnit.test("sidebar channels should be ordered case insensitive alphabetically",
         { name: "Xyz" },
         { name: "abc" },
         { name: "Abc" },
+        { name: "Est" },
         { name: "Xyz" },
+        { name: "Équipe" },
+        { name: "époque" },
     ]);
     const { openDiscuss } = await start();
     openDiscuss();
-    await contains(
-        ".o-mail-DiscussSidebarChannel:contains(abc) ~ .o-mail-DiscussSidebarChannel:contains(Abc) ~ .o-mail-DiscussSidebarChannel:contains(Xyz) ~ .o-mail-DiscussSidebarChannel:contains(Xyz)"
-    );
+    await contains(".o-mail-DiscussSidebarChannel", {
+        text: "abc",
+        before: [".o-mail-DiscussSidebarChannel", { text: "Abc" }],
+    });
+    await contains(".o-mail-DiscussSidebarChannel", {
+        text: "Abc",
+        before: [".o-mail-DiscussSidebarChannel", { text: "époque" }],
+    });
+    await contains(".o-mail-DiscussSidebarChannel", {
+        text: "époque",
+        before: [".o-mail-DiscussSidebarChannel", { text: "Équipe" }],
+    });
+    await contains(".o-mail-DiscussSidebarChannel", {
+        text: "Équipe",
+        before: [".o-mail-DiscussSidebarChannel", { text: "Est" }],
+    });
+    await contains(".o-mail-DiscussSidebarChannel", {
+        text: "Est",
+        before: [".o-mail-DiscussSidebarChannel", { text: "Xyz", count: 2 }],
+    });
```

Comment:
I don't see a change of behaviour. I suppose you added them just in case?

Reply:
Just (better) coverage for the `unnaccent` part of the sort while I was at it.